endeavouros-backgrounds-classic/PKGBUILD

```diff
@@ -0,0 +1,28 @@
+# Original Source: https://github.com/endeavouros-team/PKGBUILDS/blob/master/endeavouros-theming/PKGBUILD
+# Maintainer: LeSnake04 (dev.lesnake@gmail.com)
+pkgname=endeavouros-backgrounds-classic
+pkgver=6.4.1
+pkgrel=1
+pkgdesc='old EndeavourOS background images'
+arch=('any')
+url='https://www.endeavouros.com'
+license=('GPL3')
+
+source=(
+  "${pkgname}-${pkgver}.tar.gz::https://github.com/lesnake04/${pkgname}/archive/${pkgver}.tar.gz"
```

Comment:
wrong source url

Reply:
I removed this file already since this pull request is only for the grub theme